px/lg/templates/configmap.yaml

```diff
@@ -6,6 +6,15 @@ metadata:
     name: {{ $lg }}
 data:
     "lg_config.cfg": |
-{{ include "lg_conf" $lg_config | indent 8 }}
+        DEBUG = False
+        LOG_FILE="/var/log/lg.log"
+        LOG_LEVEL="INFO"
+
+        BIND_IP = "0.0.0.0"
+        BIND_PORT = 80
```

Comment:
Do we have any rules regarding unencrypted HTTP traffic?

Reply:
This is just the service that is serving unencrypted. The ingress does https.

Reply:
Yeap, i am aware. Just wondering if it goes against any policy that enforces any traffic (even inside) to be encrypted (it really doesn't matter much since this is the LG anyway). Just asking :)

Reply:
For pod to pod encryption we would usually use linkerd if credentials are transmitted/requried. I am unaware of any policy from ingress to service head end.

Reply:
Ack, just making sure. Thanks!